drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -1280,7 +1280,7 @@ void RasterizerStorageGLES2::sky_set_texture(RID p_sky, RID p_panorama, int p_ra
 
 			glActiveTexture(GL_TEXTURE0);
 			glBindTexture(GL_TEXTURE_CUBE_MAP, sky->radiance);
-			shaders.cubemap_filter.set_conditional(CubemapFilterShaderGLES2::USE_SOURCE_PANORAMA, false);
+			//shaders.cubemap_filter.set_conditional(CubemapFilterShaderGLES2::USE_SOURCE_PANORAMA, false);
```

Comment:
This will cause issues on hardware that is strictly compliant with OpenGL spec. By commenting out this conditional, the shader will expect to be given a ``sampler2D`` panorama texture. But you are still binding a cubemap to texture slot 0. I'm surprised this worked on your sample hardware at all.

Reply:
Oh oops, looks like I shouldn't create PRs when tired. Working on a fix...